firmware/quadruna/VC/src/tasks.c

```diff
@@ -80,6 +90,29 @@ static const CanConfig can_config = {
     .rx_overflow_clear_callback = canRxQueueOverflowClearCallback,
 };
 
+static bool dummyFilter(uint32_t msg_id)
+{
+    return true;
+}
+
+static uint32_t overflow_count = 0;
+static uint32_t read_count     = 0;
+static uint32_t write_count    = 0;
+
+static void dummyOverflow(uint32_t f)
+{
+    overflow_count++;
+}
```

Comment:
dummy functions?

Reply:
I can change the function name, but I think the behavior will not change, since I don't what should we filter and what to do when overflow.